Source/Methods/RFmxDPD/RFmxDPD.csproj

```diff
@@ -18,7 +18,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\..\..\..\..\bin\Debug\</OutputPath>

```

Comment:
Why did this file need to change? This indeed looks wrong but I'm curious.

Reply:
I got error when compiling debug. The VS complained about not able to copy the assembly to the destination, something like that. Then I started to look into where the destination is. And my source location cannot go up 6 levels of folders. If I have more than 6 levels, it is still not niceto create temp folder outside the git repo. I have seen bin or obj folder in my root directory created by other projects :)